### PR TITLE
Update integrations flow

### LINKED ISSUE #1486: Update old references to integrations page
References to the previous integrations page in the web dashboard should be updated to accommodate Workspaces UX flow in the web dashboard.

An example here with reference links to the Integrations page and Integrations dashboard:

https://learning.getpostman.com/docs/postman/postman_api/intro_api/

### PR DESCRIPTION
**Pull request by SueSmith**
Fixes #1486 

@loopDelicious have updated the postman api key setup process and intro to integrations, other than that it's just updated references to the old page

### CODE REVIEW

File: src/pages/docs/integrations/victorops.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ You can enter your own key and select a team for which the key is applicable.
 
 ## Configuring Postman Monitors
 
-In the Integrations section of [your workspaces](https://go.postman.co/workspaces) page, find VictorOps from a list of Postman’s third party integrations for Postman users.
+In the Integrations section of [your workspace](https://go.postman.co/workspaces), find VictorOps from the list of third party integrations.
```
Comment (claireriley) [inline src/pages/docs/integrations/victorops.md]: > find VictorOps from the list of 

"from" -> "in" (may be a UK v US english thing, "in" just sounds more natural to me)

File: src/pages/docs/postman/postman-api/intro-api.md [hunk 2]
```diff
@@ -12,48 +12,64 @@ contextual_links:
...
-    url: "/docs/postman/scripts/intro-to-scripts/"
+    name: "Continuous Integration"
+    url: "/docs/postman/postman-api/continuous-integration/"
 
 warning: false
 
 ---
 
-The [Postman API](https://docs.api.getpostman.com/) has several endpoints to help you integrate Postman  with your development toolchain.
+The [Postman API](https://docs.api.getpostman.com/) endpoints endpoints to help you integrate Postman within your development toolchain.
```
Comment (claireriley) [inline src/pages/docs/postman/postman-api/intro-api.md]: duplicate "endpoints"!

File: src/pages/docs/postman/postman-api/intro-api.md [hunk 3]
```diff
@@ -12,48 +12,64 @@ contextual_links:
...
 
-You can add new collections, update existing collections, update environments, or add and run monitors directly through the API. This API enables you to programmatically access your data stored in your Postman account with ease.
+You can add new collections, update existing collections, update environments, or add and run monitors directly through the API. This enables you to programmatically access data stored in your Postman account.
```
Comment (claireriley) [inline src/pages/docs/postman/postman-api/intro-api.md]: wondering if "or" could be "and" but it's a bit awkward

File: src/pages/docs/postman/postman-api/intro-api.md [hunk 4]
```diff
@@ -12,48 +12,64 @@ contextual_links:
...
 
-The easiest way to start with the API is to click the **Run in Postman** button at the top of the [Postman API documentation page](https://docs.api.getpostman.com/) and use the Postman app to send requests.
+You can get started with the API via the **Run in Postman** button at the top of the [Postman API documentation](https://docs.api.getpostman.com/) and use the Postman app to send requests.
 
-## Postman API overview
+You will need an [API key](#generating-a-postman-api-key) to access Postman API.
```
Comment (claireriley) [inline src/pages/docs/postman/postman-api/intro-api.md]: "to access Postman API" -> "to access the Postman API"

File: src/pages/docs/postman/postman-api/intro-api.md [hunk 5]
```diff
@@ -12,48 +12,64 @@ contextual_links:
...
+> The API is [rate limited](#rate-limits).
 
-1. You need a valid API Key to send requests to the API endpoints. Postman users can get a key in the [Integrations](https://app.getpostman.com/dashboard/integrations) page.
+## Generating a Postman API key
 
-1. The API is rate limited.
+You need a valid API Key to send requests to the Postman API endpoints.
 
-1. Using the API, you can add and update collections, environments, and users. You can also run monitors, create a mock server, and so much more.
+Navigate to the [web dashboard](https://app.getpostman.com/). Select a workspace and open __Integrations__. Browse the integrations—the Postman API will appear in the list. Click __View Details__.
```
Comment (claireriley) [inline src/pages/docs/postman/postman-api/intro-api.md]: Maybe

> Select a workspace and open __Integrations__. Locate the Postman API and click __View Details__.

File: src/pages/docs/postman/postman-api/intro-api.md [hunk 6]
```diff
@@ -12,48 +12,64 @@ contextual_links:
...
-[![get API key](https://assets.postman.com/postman-docs/WS-postmanAPI-apiKey.png)](https://assets.postman.com/postman-docs/WS-postmanAPI-apiKey.png)
+Use __API Key Settings__ to specify expiration periods for your keys.
 
-An API Key tells the Postman API server that the request came from you. You can get any data you have permission to access in Postman with an API Key that is generated by you.
+<img src="https://assets.postman.com/postman-docs/api-key-expire.jpg" width="300px" alt="API Key Expires"/>
 
-For ease of use inside Postman, you could store your API key in an [environment variable](/docs/postman/variables-and-environments/variables/) called ``postman-api-key``. This collection automatically uses this key to make API calls.
+### Authentication
+
+You will need to authenticate your requests to Postman API by sending your API Key in the `X-Api-Key` header of every request you make.
```
Comment (claireriley) [inline src/pages/docs/postman/postman-api/intro-api.md]: "to Postman API" I'd change to "to the Postman API"

**Comment by SueSmith** [comment]
Thanks @claireriley! Think I've done them all.

**Comment by ch264** [comment]
@claireriley @SueSmith @loopDelicious what is the status on this PR? It looks like the changes are outdated?

**Comment by SueSmith** [comment]
k we should be updated, @claireriley if those changes look ok and you can approve we should be able to just publish this

**Pull request merged**